spec/models/endpoint_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Endpoint, type: :model do
+  context 'when creating an endpoint' do
+    subject { described_class.new }
+
+    it 'requires a kimono_id' do
+      expect { subject.save! }.to raise_error
+    end
+  end
+
+  context 'with an established endpoint' do
```

Comment:
Yeah, I like that wording better here too.  Initially I had a slightly different implementation that did not require a kimono_id in the initializer.  So my brain was kind of hurting about the difference between an instance (w/ or w/o an id) versus... one that definitely had one.  That is now a useless distinction, and gladly so :)